servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -214,48 +306,167 @@ public Duration maxEjectionTimeJitter() {
-
-        private int enforcingSuccessRate = 100;
-
-        private int successRateMinimumHosts = 5;
-
-        private int successRateRequestVolume = 100;
-
-        private int successRateStdevFactor = 1900;
-
-        private int failurePercentageThreshold = 85;
-
-        private int enforcingFailurePercentage;
+    public Builder toBuilder() {
+        return new Builder(
+                ewmaHalfLife, failedConnectionsThreshold, intervalJitter,
+                serviceDiscoveryResubscribeInterval, serviceDiscoveryResubscribeJitter,
+                consecutive5xx, interval, baseEjectionTime,
+                maxEjectionPercentage, enforcingConsecutive5xx,
+                enforcingSuccessRate, successRateMinimumHosts,
+                successRateRequestVolume, successRateStdevFactor,
+                failurePercentageThreshold, enforcingFailurePercentage,
+                failurePercentageMinimumHosts, failurePercentageRequestVolume,
+                maxEjectionTime, maxEjectionTimeJitter);
+    }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        OutlierDetectorConfig that = (OutlierDetectorConfig) o;
+        return ewmaCancellationPenalty == that.ewmaCancellationPenalty &&
+                ewmaErrorPenalty == that.ewmaErrorPenalty &&
+                failedConnectionsThreshold == that.failedConnectionsThreshold &&
+                consecutive5xx == that.consecutive5xx &&
+                maxEjectionPercentage == that.maxEjectionPercentage &&
+                enforcingConsecutive5xx == that.enforcingConsecutive5xx &&
+                enforcingSuccessRate == that.enforcingSuccessRate &&
+                successRateMinimumHosts == that.successRateMinimumHosts &&
+                successRateRequestVolume == that.successRateRequestVolume &&
+                successRateStdevFactor == that.successRateStdevFactor &&
+                failurePercentageThreshold == that.failurePercentageThreshold &&
+                enforcingFailurePercentage == that.enforcingFailurePercentage &&
+                failurePercentageMinimumHosts == that.failurePercentageMinimumHosts &&
+                failurePercentageRequestVolume == that.failurePercentageRequestVolume &&
+                intervalJitter.equals(that.intervalJitter) &&
+                ewmaHalfLife.equals(that.ewmaHalfLife) &&
+                interval.equals(that.interval) &&
+                baseEjectionTime.equals(that.baseEjectionTime) &&
+                maxEjectionTime.equals(that.maxEjectionTime) &&
+                maxEjectionTimeJitter.equals(that.maxEjectionTimeJitter);
+    }
 
-        private int failurePercentageMinimumHosts = 5;
+    @Override
+    public int hashCode() {
+        return Objects.hash(ewmaHalfLife, ewmaCancellationPenalty, ewmaErrorPenalty, failedConnectionsThreshold,
```

Comment:
Is there a use-case that requires equals/hashCode for `OutlierDetectorConfig`? If no, consider removing these overrides. If yes and it could be frequest, consider using other generation option that does not use `Objects.hash` because it creates an array for varargs.

Reply:
Only that since it's simple data it makes sense. I can remove them though: they are likely not good map keys or often directly compared.